src/components/SVGViewer/SVGViewer.tsx

```diff
@@ -600,16 +599,45 @@ export class SVGViewer extends Component<SvgViewerProps, SvgViewerState> {
     }
   };
 
-  zoomOnCurrentAsset = (currentAsset: Element | null) => {
+  // used to overcome the issue when an element is in the DOM
+  // but its position changes over time, so you can't use it until it's stable
+  getStableBoundingClientRect = (
+    el: Element,
+    maxAttempts = 10
```

Comment:
I'm prob just missing how this is happening but where are you saying: "ok stop doing this function when maxAttempts has reached 0"?

Reply:
A bit down there https://github.com/cognitedata/gearbox.js/pull/733/files#diff-23b593734827286577a091e3c2f550964f5853adf83e063e3cea92a7a6b6a701R618-R619

I actually never seen it executed more than 2-4 times, but added it just in case

Reply:
ah yes thanks, gotcha